### DISCUSSION TITLE
Criteria for GSoC Mentors

### DISCUSSION THREAD

**Discussion by adityasaky** [cncf/mentoring#888] [Q&A] [answered]
Can a community member be a primary mentor with input from a maintainer as a secondary mentor?

**Comment by aliok** [comment, marked as answer]
Short answer: Yes

Long answer: The important thing is that the mentee is assisted. If the secondary mentor you mentioned above trusts in the ability of the primary mentor, then there would be no objections from GSoC admins. The secondary mentor would need to understand that they might also mentor the primary mentor.

**Reply by adityasaky** [reply]
Got it, thanks!
